packages/flutter_tools/lib/src/flutter_manifest.dart

```diff
@@ -858,6 +942,17 @@ final class AssetTransformerEntry {
   final String package;
   final List<String>? args;
 
+  Map<String, Object?> get descriptor {
+    return <String, Object?>{
+      _kPackage: package,
+      if (args != null)
+        _kArgs: args,
+    };
+  }
+
+  static const String _kPackage = 'package';
+  static const String _kArgs = 'args';
```

Comment:
Good catch. I had originally updated all of `flutter_manifest.dart` to use constants rather than raw strings, but broke something somewhere in the 1000 line diff and just reverted it. I'll update these in a follow up.